05_syntax_lab/04.py

```diff
@@ -1,8 +1,22 @@
-"""
-Write a program that reads lines from the user
-until an empty line is inserted.
-After the user typed in an empty line,
-print all previously inserted lines in reverse
-order (from last to first)
-"""
-
+"""
+Elad Agam
+04 - Read lines untill getting an empty lines
+Print all the input line by line - backwards
+"""
+
+Input = []
+while True:
+	print "Enter a line"
+	line = raw_input()
+	if not line == "":
+		Input.append(line)
+	else:
+		break
+
+
+numoflines =(len(Input)) 
+for i in range(numoflines):
+	print Input.pop()
+
+
+

```

Comment:
Nice! Especially the use of lists.
It would be better to use all-lower-case variable names but not ```input``` as that is a reserved word. Maybe ```line``` ?
It  might be better to print the lines without destroying the Input. Can you do that?